src/widgets/MainTask/MainTask.test.tsx

```diff
@@ -12,30 +12,48 @@ import '@testing-library/jest-dom/extend-expect';
 
 jest.mock('shared/lib/helpers');
 
-const state = { tasks: { data: [{ ...testTask }] } };
-
-test('expands and collapses the accordion when clicked', async () => {
-    (getDateAndTime as jest.Mock).mockImplementation(() => '14.11.2023 / 15.53.23');
-    render(
-        <Provider store={setupStore(state)}>
+describe('MainTask test', () => {
+    const renderMainTask = () => render(
+        <Provider store={setupStore({ tasks: { data: [testTask] } })}>
             <MainTask task={testTask} />
         </Provider>,
     );
 
-    const taskTitle = screen.getByTestId('TaskCard-title');
-    expect(taskTitle).toBeInTheDocument();
-    expect(screen.getByText('Created:14.11.2023 / 15.53.23')).toBeInTheDocument();
+    beforeEach(() => {
+        (getDateAndTime as jest.Mock).mockImplementation(() => '14.11.2023 / 15.53.23');
```

Comment:
`14.11.2023 / 15.53.23` better to move to a const

Reply:
Done